serverless.yml

```diff
@@ -88,7 +88,7 @@ resources:
             KeyType: "HASH" 
         BillingMode: PAY_PER_REQUEST
         TableName: ${self:provider.environment.GUESTS_TABLE}
-    ResponsesDynamoDbTable:
+    ResponseCollectionDynamoDbTable:
```

Comment:
Any reason you changed the table name? Typically you don't need `Collection` in the table name since it being in a database already implies it is a collection of _something_.

Reply:
I deleted the old dynamo table because it didn't have relevant data anymore and for some reason when I did `sls deploy` it didn't recreate the table, so I changed the name to make a new table and get it working. Now I'm thinking that the deletion of the table might have just taken an abnormally long amount of time. I'll try creating it again.